client/src/js/app/Main.js

```diff
@@ -74,15 +73,15 @@ export const Main = ({ ready, onLoad }) => {
 
                 <DevDialog />
                 <UploadOverlay />
-            </div>
+            </React.Fragment>
         );
     }
 
     return <LoadingPlaceholder />;
 };
 
-export const mapStateToProps = state => ({
-    ready: state.account.ready && Boolean(keys(state.settings).length)
+export const mapStateToProps = ({ account: { ready }, settings: { data } }) => ({
+    ready: ready && Boolean(data)
 });
```

Comment:
I don't find this readable. Stick to the signature of `mapStateToProps` used everywhere else in Virtool.
```suggestion
export const mapStateToProps = state => ({
    ready: state.account.ready && Boolean(state.settings.data)
});
```